### DISCUSSION TITLE
saf-training/resources/02

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#265] [Ideas]
# saf-training/resources/02

Using Codespaces for a Lab Environment You can follow along with each exercise given in these training classes by creating a GitHub Codespace from one of the MITRE SAF team's Gi...

https://mitre.github.io/saf-training/resources/02.html

**Comment by karikarshivani** [comment]
Add a note that `Copy the main only` is optional to check

**Comment by p-oneil** [comment]
Copied to PR
